Cross.toml

```diff
@@ -0,0 +1,2 @@
+[target.x86_64-unknown-freebsd]
+image = "docker.io/rustembedded/cross:x86_64-unknown-freebsd"
```

Comment:
I'm not sure this is the same as .cargo/config.toml, but it might be. Maybe move that to https://github.com/MindFlavor/prometheus_wireguard_exporter/blob/master/.cargo/config.toml?

Would it be possible that the existing .cargo.config.yml conflicts with e.g. the armv7 build you're trying to achieve? 🤔